video/vdu.h

```diff
@@ -22,8 +23,45 @@ void VDUStreamProcessor::vdu(uint8_t c) {
 	if (consoleMode) {
 		DBGSerial.write(c);
 	}
-	
+
+	if (printerOn) {
+		switch(c) {
+			case 0x03:
+				printerOn = false;
+				break;
+			// case 0x09:	// translate "cursor right" to a space, as terminals treat character 9 as tab
+			// 	vdu_print(32);
+			// 	break;
```

Comment:
I'd noticed when using the arrow keys to edit a command that the terminal output would be "wonky" and not match what I had on screen, and worked out that this was because the "cursor right" is character 9, which is TAB on the attached terminal.
this hard-coded swap of character 9 to 32 fixes the display issue I saw, however it would prevent printing of the TAB character to the "printer" - I _think_ we're better off leaving character 9 alone